sphinx/apidoc.py

```diff
@@ -194,7 +195,14 @@ def shall_skip(module, opts):
 
     # skip it if there is nothing (or just \n or \r\n) in the file
     if path.exists(module) and path.getsize(module) <= 2:
-        return True
+        skip = True
+        if module.endswith('__init__.py'):
```

Comment:
Please use `os.path.basename(module) == '__init__.py'`. `endswith` is not good.

Reply:
Done